lib/plugins/create/templates/aws-nodejs-typescript/webpack.config.js

```diff
@@ -3,6 +3,11 @@ const slsw = require('serverless-webpack');
 const nodeExternals = require('webpack-node-externals');
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 
+/*
+Commenting out this line will cause the ts compilation of the serverless.ts file to fail
+ */
+delete process.env.TS_NODE_PROJECT;
```

Comment:
What's the value of this env before removal ? I don't understand how it impacts your ts config.

Reply:
This is only required if the there isn't a `tsconfig.json` in the project eg `tsconfig.app.json`.

**Background**:
`TS_CONFIG_PROJECT` is used by ts-node to specify the tsconfig file if it's required to be different from the default `tsconfig.json`.  However, setting it so `serverless.ts` compiles, will prevent the project code from compiling, hence the need to remove the property value in the webpack config.

**Reproduction**
Rename `tsconfig.json` to `tsconfig.app.json`, comment out line 9 and run `TS_CONFIG_PROJECT=./tsconfig.app.json npx sls webpack`.

You should see something like:
```shell
Failed to load ./tsconfig.paths.json: Missing baseUrl in compilerOptions
Serverless: Bundling with Webpack...
tsconfig-paths-webpack-plugin: Found no baseUrl in tsconfig.json, not applying tsconfig-paths-webpack-plugin
```
Uncomment line 9 and rerun, and the serverless.ts and application code should compile, package properly

**Use case**
I have a [nx](https://nx.dev) monorepo that uses multiple tsconfig files within the application.  The one to compile the code is `tsconfig.app.json` (there's another for jest unit tests) which extends `tsconfig.json` which in turn extends the `tsconfig.base.json` (similar to the `tsconfig.paths.json` here) which specifies the paths to other libraries within the project.  
If I don't specify `TS_CONFIG_PROJECT=tsconfig.app.json` then the `serverless.ts` doesn't compile, if I do, the application code doesn't unless I unset the environment variable in the webpack config 

(I suspect the error is due to `ts-node` not respecting/understanding the 'extended' config property, but I've not confirmed that's the case).

**Future steps**
Generate a  nx monorepo with serverless webpack.  I currently have a company repo that I can't share, but shouldn't be too difficult to create an additional serverless template example with this use case in it